Website edits

nwlandry opened the pull request:
This PR
* Fixes the color switching issues
* Removes unnecessary files
* Fixes sphinx errors from the migration to the PyData theme
* Fixes #341

maximelucas commented on xgi/convert/incidence.py:
```diff
@@ -37,7 +37,7 @@ def from_incidence_matrix(d, create_using=None, nodelabels=None, edgelabels=None
     See Also
     --------
-    incidence_matrix
+    ~xgi.linalg.hypergraph_matrix.incidence_matrix
     to_incidence_matrix
```
how come the first line needs the long reference and the second one does not?

nwlandry replied:
You need the long reference if it's not in the same file.

maximelucas commented on docs/source/user_guides.rst:
```diff
@@ -55,13 +51,15 @@ User Guides
     	+++
 
-        .. button-ref:: indepth_tutorials
+        .. button-ref:: api/tutorials/in_depth
```
Now you're using the path to the rst file rather than the anchor right? 
Is it better practice or what's idea behind it?

nwlandry replied:
Hard to know what best practices are. But, FWIW, matplotlib does it with relative paths: https://github.com/matplotlib/matplotlib/blob/main/doc%2Findex.rst

maximelucas replied:
Probably easier to track back the file it looks to this way

maximelucas commented on docs/source/installing_more.rst:
```diff
@@ -1,8 +1,8 @@
-.. _installing_more:
+:orphan:
```
what does :orphan: does? better than having nothing?

nwlandry replied:
:orphan: suppresses the "document is not part of any toctree" in sphinx. This is the only solution I found to that problem.

maximelucas commented on docs/source/installing.rst:
```diff
@@ -1,8 +1,6 @@
-Installing
-***************
+**************
+Installing XGI
```
I feel like adding "XGI" makes it longer and the menu doesn't have so much space, so I think I wouldn't add it

nwlandry replied:
Sounds good. I'll change it back.

nwlandry replied:
Fixed.

maximelucas commented:
Nice work Nich! Just a few questions. I like the color switch and that you added the case studies back in.

nwlandry commented:
> Nice work Nich! Just a few questions. I like the color switch and that you added the case studies back in. 

Thanks so much for the review! I believe I addressed all your comments.
